src/main/java/org/fenixedu/academic/domain/Professorship.java

```diff
@@ -45,6 +47,9 @@
     public static final Comparator<Professorship> COMPARATOR_BY_PERSON_NAME = new BeanComparator("person.name",
             Collator.getInstance());
 
+    public static final String PROFESSORSHIP_CHANGED = "academic.professorship.change";
+    public static final String PROFESSORSHIP_DELETED = "academic.professorship.deleted";
```

Comment:
This is declared but then it is never used.
Are we missing a Signal.emit() somewhere, or is the constant no necessary?